botprise/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # CHANGELOG - Botprise Integration
 
-1.0.0 / Initial Setup instructions
-==================
-### Changes
+## 1.0.0 / Initial Setup instructions
```

Comment:
Can you find the date of the merge commit and update this header? 
While I know we're not necessarily standardizing the date with these PRs, its really weird that this isn't a date 🙂